releases/4.2.11/consumer-setup-4.2.11.input

```diff
@@ -0,0 +1,19 @@
+
+
+2
+tempZone
+4211_irods-catalog-provider_1
```

Comment:
This will not allow us to run things in parallel (at least not flexibly, like the rest of the testing environment). I think we might just need to treat these projects like one of the others and run setup when the container is stood up. We would then only need to skip installing the packages. So... gonna need to rethink this a little bit.